libclc/generic/lib/gen_convert.py

```diff
@@ -26,6 +27,12 @@
 #
 # convert_<destTypen><_sat><_roundingMode>(<sourceTypen>)
 
+import sys
+
+clspv = False
+if len(sys.argv) == 2 and sys.argv[1] == "--clspv":
+    clspv = True
+
```

Comment:
```suggestion
import argparse

parser = argparse.ArgumentParser()
parser.add_argument('--clspv', help="Generate the clspv variant of the code")
args = parser.parse_args()
clspv = args.clspv
```
It's not that much more code and will be easier to maintain, use, and extend.

Reply:
I wanted to avoid the dependency on argparse to be added. But Python version not having it by default are getting very old and unsupported so I guess it's definitely better to use it.